docs/sources/enterprise/configuration.md

```diff
@@ -0,0 +1,187 @@
++++
```

Comment:
I just noticed the title. Please change it to enterprise-configuration.md. It will be less likely to get confused with the other configuration.md file we have.